iree/turbine/aot/compiled_module.py

```diff
@@ -207,6 +240,21 @@ def globals_defs(self) -> Generator[Tuple[str, GlobalsDef], None, None]:
         )  # type: ignore
 
     def def_attribute(self, key, value):
+        if isinstance(value, ExportTargetDef):
+            if isinstance(value.target, ExportedProgram):
+                value = ExportedProgramDef(
```

Comment:
Shouldn't this be returned as well? It is setting the value but this falls through. I see it is handled further down, if so we should handle it right before the follow up case.

Reply:
I moved this case down before the handling of `ExportedProgramDef`.